i18n/fi/design-system.json

```diff
@@ -162,6 +162,7 @@
   "wds-avatar-badges-staff-tooltip": "FANDOM-henkilökunta", 
   "wds-avatar-badges-vstf-tooltip": "VSTF", 
   "wds-fandom-homepage-aria-label": "Fandom homepage", 
-  "wds-gamepedia-homepage-aria-label": "Gamepedia homepage", 
-  "wds-wikiaorg-homepage-aria-label": "Wikia.org homepage"
-}
\ No newline at end of file
+  "wds-gamepedia-homepage-aria-label": "Gamepedia homepage",
+  "wds-wikiaorg-homepage-aria-label": "Wikia.org homepage",
+  "username-anonymous": "A FANDOM User"
```

Comment:
These should use "Fandom" instead but we have to be careful if manually changing them, as I don't know how that works for non-Latin alphabets like Russian and the Asian languages.

Reply:
I'm going to remove this for now and handle translations in a separate ticket.